### ISSUE TITLE
Successful attribute option POST returns 'true', not the option hash or value

### ISSUE THREAD

**Issue by dnadle** [magento/magento2#6300]
### Preconditions

1. Magento 2.1.0.
### Steps to reproduce

1. POST an new option value using /V1/products/attributes/{attributeCode}/options. Request body looks like `{"option": {"label": "FOOBAR"} }`
2. Examine response content.
### Expected result

1. Returned body should contain full record of option inserted including the option "value."
### Actual result

1. Response body contains only 'true'. To learn the value of the newly inserted option you most GET the entire list again.

**Comment by mbrinton01** [comment]
@dnadle please provide full POST call used. Thanks!

**Comment by dnadle** [comment]
The full post body is

```
{ "option":
    {
        "label": "FOOBAR",
        "is_default": false
    }
}
```

**Comment by mbrinton01** [comment]
Thank you @dnadle, I have confirmed this behavior.  Tracking internally as a bug under https://jira.corp.magento.com/browse/MAGETWO-57585.

**Comment by dnadle** [comment]
I'd appreciate it if you could look at [#6127](https://github.com/magento/magento2/issues/6127) and [#6120](https://github.com/magento/magento2/issues/6120) as well. These bugs make it very difficult to manage products through the API.

**Comment by magento-engcom-team** [comment]
@dnadle, thank you for your report.
We've created internal ticket(s) MAGETWO-57585 to track progress on the issue.

**Comment by kraputa-sintra** [comment]
#distributed-cd

**Comment by k-raputa** [comment]
#distributed-cd

**Comment by magento-engcom-team** [comment]
@kraputa-sintra thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by magento-engcom-team** [comment]
@k-raputa thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by magento-engcom-team** [comment]
Hi @milindsingh. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by milindsingh** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @milindsingh. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @milindsingh, here is your [Magento instance](https://i-6300-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-6300-2-3-develop.instances.magento-community.engineering/admin](https://i-6300-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by milindsingh** [comment]
@dnadle @mbrinton01 I rechecked and found that

+ 2.3-develop : the API response was `"id_FOOBAR"`
+ 2.2-develop:  the API response was `"true"`

What do you suggest, 
1. We shall use the same object as in return i.e. 
```
{ 
   "option":
       {
           "label": "FOOBAR",
           "is_default": false
       }
}
```

2. Or just backport the of 2.3-develop as the return, i.e
```
"id_FOOBAR"
```

**Comment by mbrinton01** [comment]
@milindsingh Option 2 (backport 2.3-develop approach)

**Comment by ghost** [comment]
Hi @dnadle thank you for you report, this issue has already fixed in 2.3-develop branch. https://github.com/magento/magento2/pull/19108
